src/server/deamon/index.ts

```diff
@@ -29,19 +30,23 @@ export interface Device {
   platform: string;
 }
 
+type ResponseCallback = (...args: any[]) => void;
+
 const selectedDeviceIdKey = 'selectedDeviceId';
 
 export class DaemonServer extends Dispose {
   private process?: ChildProcess;
-
-  private eventHandlers: { [key: string]: (params?: { [key: string]: any }) => void } = {};
+  private eventHandlers: Record<string, (params?: { [key: string]: any }) => void> = {};
```

Comment:
Damn that’s so much nicer I’ll be using that in the future instead :)

Reply:
Btw, inst the params also a Record?

Reply:
ah missing it